csharp_dotnetcore/1.Console-EchoBot/Console-EchoBot.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <CodeAnalysisRuleSet>Console-EchoBot.ruleset</CodeAnalysisRuleSet>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
+    <CodeAnalysisRuleSet>Console-EchoBot.ruleset</CodeAnalysisRuleSet>
```

Comment:
Both of these XMLElements can be deleted. It's since become clear they're only for the old-school CodeAnalysis tools, and the newer stuff just doesn't need them.

Reply:
We actually just collapsed this (not conditional on Debug | Release).  Turns out it seems like we need the line..